### DISCUSSION TITLE
Joomla

### DISCUSSION THREAD

**Discussion by ghost** [HuubMons/HuMo-genealogy#27] [Q&A] [answered]
Does JHTML still exist in joomla 4.x?

I only see it in the 2.5 doc but I'm not a joomla specialist.

**Comment by HuubMons** [comment, marked as answer]
About the Joomla component:

In 2011 I made a HuMo-genealogy Joomla component. This component uses the special CMS variables.
But: this component is no longer needed. Yossi made some changes in HuMo-genealogy, so I removed the Joomla component a few years ago from the website.

Because of backward compatibility the component was still supported. It's possible this Joomla component isn't used anymore.
So I don't mind if these items are removed.

**Reply by ghost** [reply]
Ok I will delete it.
